test/FpF/FpFComparisonOperators.cpp

```diff
@@ -0,0 +1,89 @@
+//!
```

Comment:
It would be worthwhile adding some comparison tests that show large and extremely large decimal values